### DISCUSSION TITLE
限定共有記事の「編集」が見えない

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#882] [要望・アイデア・不具合]
いつもお世話になります。

少し前から、限定共有記事の「編集」の文字が見えなくなりました。
ぎりクリックはできるのですが、メニューの表示位置をもう少し下げていただければと思います。
（ベータ版が影響している？）

![menu](https://github.com/user-attachments/assets/151103f3-3d1c-403a-865b-c1774eb88900)

よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@nak435 
ご報告ありがとうございます！ご不便をおかけして申し訳ありません。
弊社で確認したところ再現できなかったため、調査を進めるにあたり下記について教えていただけると幸いです。

## 教えていただきたいこと
- 広告ブロックの拡張機能はお使いではないでしょうか？
    - スクリーンショットを拝見したところ、ドロップダウンを表示するボタンの上に本来表示されているはずのSNSのシェアボタン3つが表示されていないようでしたので、念のためご確認をお願いします。
    ![スクリーンショット](https://github.com/user-attachments/assets/872132c5-a0bd-465f-9f76-ae7075d844fd)

- お使いのOS・ブラウザについて、種類とバージョンを教えてください

ご協力のほど、よろしくお願いいたします :bow:

**Reply by nak435** [reply]
> 広告ブロックの拡張機能はお使いではないでしょうか？

使っていました。外すと、SNSアイコンが出てきました。
これが原因ですね。失礼しました。


> お使いのOS・ブラウザについて、種類とバージョンを教えてください

macOS Sonoma 14.5、Safari 17.5 (19618.2.12.11.6)

---

可能であれば、20〜30ピクセル表示位置を下げてもらえるとありがたいです。
よろしくお願いします。

**Reply by tanaka-asami415** [reply]
@nak435 
早速ありがとうございます！

大変恐れ入りますが、Qiitaは広告収入で運営しているため、広告ブロックご利用時の挙動は確認しておりません🙏
メニュー位置の見直しの際には今回のご意見も参考にさせていただきますが、広告ブロックを利用されていると他にも正常にサービスをご利用いただけない可能性があるため
可能でしたら、機能をオフにしていただくか、ホワイトリストにQiitaを登録してご対応いただけますと幸いです🙇

**Reply by nak435** [reply]
大変失礼しました。
Qiita利用時は広告ブロックをOFFにします。

**Reply by tanaka-asami415** [reply]
@nak435 
とんでもないです。ご理解いただきありがとうございます🙇

**Comment by nak435** [comment]
本件、クローズいたします。
